.travis.yml

```diff
@@ -17,9 +17,9 @@ language: java
 matrix:
   include:
     - jdk: "oraclejdk8"
-      env: PROFILE="-Dhadoop.version=2.7.1 -Dscala-2.11 -Pinclude-yarn-tests -Dmaven.javadoc.skip=true"
```

Comment:
I think it is bad practice because it lets you get away with broken JavaDoc. On my machine it saves 2 minutes compilation time but for CI I think we should keep it.